src/server/game/Entities/Player/Player.cpp

```diff
@@ -813,7 +813,7 @@ int32 Player::getMaxTimer(MirrorTimerType timer)
             {
                 if (!IsAlive() || HasAuraType(SPELL_AURA_WATER_BREATHING) || GetSession()->GetSecurity() >= AccountTypes(sWorld->getIntConfig(CONFIG_DISABLE_BREATHING)))
                     return DISABLED_MIRROR_TIMER;
-                int32 UnderWaterTime = 3 * MINUTE * IN_MILLISECONDS;
+                int32 UnderWaterTime = sWorld->getIntConfig(CONFIG_WATER_BREATH_TIMER) * MINUTE;
```

Comment:
looking at the conf and the world.cpp, are doing this in minutes or milliseconds? because i am seeing 3000 as the default

Reply:
Sorry, you are correct. This right now is confusing - the behavior is correct right now, but the set value is neither milliseconds or minutes. I will change config default value to 180000, which is 3 minutes in milliseconds and remove the MINUTE multiplier.